src/main/resources/templates/aeolus/java/plain_maven_blackbox.sh

```
@@ -6,36 +6,48 @@ build () {
+    sed -i "s#MAIN_CLASS#${main_class}#" testsuite/config/default.exp
   else
+    echo "no main method found. quitting the test run."
     exit 1
   fi
+}
 
-  JAVA_FLAGS="-Djdk.console=java.base"
+custom_checkers () {
+  echo '⚙️ executing custom_checkers'
+  # all java files in the assignment folder should have maximal line length 80
+  pipeline-helper line-length -l 80 -s ${studentParentWorkingDirectoryName}/ -e java
```

Shellcheck (0.10.0)

[warning] studentParentWorkingDirectoryName is referenced but not assigned.

(SC2154)